docs/sdk/src/polkadot_sdk/polkadot.rs

```diff
@@ -47,10 +47,11 @@
 //!   Chain*").
 //! * (heterogenous) Sharded Execution: Yet, each parachain is free to have its own execution logic
 //!   (runtime), which also encompasses governance and sovereignty. Moreover, Polkadot ensures the
-//!   correct execution of all parachain, without having all of its validators re-execute all
-//!   parachain blocks. When seen from this perspective, the fact that Polkadot executes different
-//!   parachains means it is a platform that has fully delivered (the holy grail of) "Full Execution
-//!   Sharding". TODO: link to approval checking article. <https://github.com/paritytech/polkadot-sdk-docs/issues/66>
+//!   correct execution of all parachains, without having all of its validators re-execute all
+//!   parachain blocks. When seen from this perspective, the fact that different Polkadot validators
+//!   execute different parachains means it is a platform that has fully delivered (the holy grail
+//!   of) "Full Execution Sharding".
+//!   Learn about this process called [Approval Checking](https://docs.google.com/presentation/d/1fjiTeT33GpxeM9TyBvxFnSVYR9NiU9SrkA2RBxSL5k8).
```

Comment:
nit but maybe better to link also the talk (is there any?)

Reply:
I couldn't found any... maybe @rphmeier knows?

Reply:
I'd suggest this comprehensive post by Rob:
https://polkadot.network/blog/polkadot-v1-0-sharding-and-economic-security/#approval-checking-and-finality